common/projectconfig/projectconfig.go

```diff
@@ -43,16 +44,43 @@ func ConfigPaths(input []string) []string {
 	if len(input) > 0 {
 		return input
 	}
-	path := GetDefaultConfigPath()
+	path, ok := GetDefaultConfigPath().Get()
+	if !ok {
+		return []string{}
+	}
 	_, err := os.Stat(path)
-	if err == nil {
-		return []string{path}
+	if err != nil {
+		return []string{}
 	}
-	return []string{}
+	return []string{path}
 }
 
-func GetDefaultConfigPath() string {
-	return filepath.Join(internal.GitRoot(""), "ftl-project.toml")
+func GetDefaultConfigPath() optional.Option[string] {
```

Comment:
nit: [Idiomatic Go](https://go.dev/doc/effective_go#Getters) is to not use "getter" style naming, I'd just call this `DefaultConfigPath()`.

Reply:
done!